linera-core/src/unit_tests/client_test_utils.rs

```diff
@@ -30,32 +30,26 @@ use std::{
 };
 use tokio::sync::oneshot;
 
-#[cfg(any(feature = "rocksdb", feature = "aws", feature = "scylladb"))]
-use linera_views::lru_caching::TEST_CACHE_SIZE;
-
 #[cfg(feature = "rocksdb")]
 use {
-    linera_storage::RocksDbStoreClient, linera_views::rocks_db::TEST_ROCKS_DB_MAX_STREAM_QUERIES,
+    linera_storage::RocksDbStoreClient, linera_views::rocks_db::create_rocks_db_common_config,
```

Comment:
Very often, public functions can be turned into static methods -- which helps with imports and code readability. Here, this one will probably become a constructor `RocksDbStoreConfig::new` if we follow #993.

Reply:
Ok, will see later about that.